packages/vue-3/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
-  "extends": "../../tsconfig",
-  "exclude": [
-    "**/node_modules",
-    "**/dist",
-    "../../demos",
-    "../vue-2"
-  ]
+  "extends": "../../tsconfig.tiptap.json",
+  "compilerOptions": {
+    "paths": {
+      "@tiptap/*": ["../*/src"]
+    }
+  },
+  "include": ["**/*.d.ts", "**/*.ts", "**/*.tsx", "**/*.vue"],
```

Comment:
Do we need those two lines (include, exclude) in every package specific tsconfig.json? Doesn't it get extended by base tsconfig?

Reply:
Unfortunately yeah - those paths are resolved when the tsconfig is loaded. I tried to put this out and move it into the extended tsconfig, but then the paths were picked from the root directory instead of the relative one.